src/index.js

```diff
@@ -1,6 +1,17 @@
 'use strict'
 
-exports.util = require('./util.js')
-exports.resolver = require('./resolver.js')
-exports.codec = exports.util.codec
-exports.defaultHashAlg = exports.util.defaultHashAlg
+const util = require('./util.js')
+
+/**
+ * @typedef {import('interface-ipld-format').Format<object>} ObjectFormat
+ */
+
+/**
+ * @type {ObjectFormat}
+ */
+module.exports = {
+  util,
+  resolver: require('./resolver.js'),
```

Comment:
requires grouped at the top would be nice

Reply:
Done!